### DISCUSSION TITLE
add C# version in steam

### DISCUSSION THREAD

**Discussion by RakibRyan** [godotengine/godot-proposals#9732] [Editor]
### Describe the project you are working on

.

### Describe the problem or limitation you are having in your project

don't know when updates are rolled out.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

.

### Is there a reason why this should be core and not an add-on in the asset library?

.

**Comment by ZeeWeasel** [comment]
Hi, I am not entirely sure if there is a single reason why not adding the c# branch onto Steam, but from a cursory search, I believe that getting the Godot C# version to run out-of-the-box (which is a thing Steam requires of it's games / apps) is still an issue, since it requires installing the .NET SDK.

https://steamcommunity.com/app/404790/discussions/0/1700542968203734309/

In case you want to track your time through Steam, a possible way to still use Steam, is to manually replace the .exe https://github.com/godotengine/godot/issues/24711

**Comment by KoBeWi** [comment]
> don't know when updates are rolled out.

There is a built-in update checker now, so you don't need to use Steam: https://github.com/godotengine/godot/pull/75916
